compiler/compiler_test.go

```diff
@@ -20,50 +19,58 @@ func requireNoError(t *testing.T, err error) {
 	}
 }
 
-func requireProtoEqual(t *testing.T, want, got proto.Message) {
-	t.Helper()
-	opts := cmp.Options{protocmp.Transform()}
-	if diff := cmp.Diff(want, got, opts); diff != "" {
-		t.Errorf("protos not equal (-want +got):\n%s", diff)
-	}
-}
-
 func requireEqual(t *testing.T, want, got interface{}) {
 	t.Helper()
-	opts := cmp.Options{protocmp.Transform()}
-	if diff := cmp.Diff(want, got, opts); diff != "" {
-		t.Errorf("not equal (-want +got):\n%s", diff)
+	var opts []cmp.Option
```

Comment:
Should we just bring in testify?

Reply:
Yes. Let me do this in a follow up PR if that's ok.